Formula/watchman.rb

```diff
@@ -78,6 +57,6 @@ def post_install
   end
 
   test do
-    assert_equal(/(\d+\.\d+\.\d+)/.match(version)[0], shell_output("#{bin}/watchman -v").chomp)
+    assert_equal(version.to_s, shell_output("#{bin}/watchman -v").chomp)
```

Comment:
Tried to turn this into a real test. Unfortunately it doesn't work with the sandbox:
```
2021-06-03T05:38:10,359: [cli] Failed to open /Users/carlocab/Library/LaunchAgents/com.github.facebook.watchman.plist for write: Operation not permitted
```
See also https://github.com/Homebrew/legacy-homebrew/pull/43972.